### DISCUSSION TITLE
Including the same text in multiple places

### DISCUSSION THREAD

**Discussion by hughjonesd** [GitbookIO/community#493] [0. General bugs and questions]
I'd like to include the same chunk of text in more than one place (and keep it in sync, i.e. update it from one location). Is that possible? I know there are external embeds.

**Comment by gregberge** [comment]
Hi @hughjonesd, as of today it's not possible but it could be soon. I have some questions:

- What kind of text would you like to add in this block?
- How often would you like to be able to edit this block?
- Would you like to be able to make change request on a sync content?

**Reply by hughjonesd** [reply]
> 
> - What kind of text would you like to add in this block?
> 

I think the same as elsewhere - bullet points, links, whatever.  

A use case might be a set of principles or a company mission statement. Or maybe a function signature or similar. 

> - How often would you like to be able to edit this block?
> 

Again, same as for everything else. Being updatable is the point - if you don't need this, then you can copy and paste. 


> - Would you like to be able to make change request on a sync content?

Ideally yes. If it's in multiple places, it's possibly important text. 

What these answers point to, I guess, is the ability to "include" an arbitrary block - or blocks - in multiple locations.

**Comment by JaniceManwiller** [comment]
This could also include variable values such as the product or company name, to ensure that it's used correctly throughout the doc.

**Comment by saurav-datta** [comment]
Here is how the feature works in [Obsidian](https://www.ctnet.co.uk/embedding-one-obsidian-note-inside-another-obsidian-note/#:~:text=You%20can%20embed%20a%20single,to%20embed%20from%20the%20list)
